db/migrate/20170918125937_ids_to_bigints20.rb

```diff
@@ -0,0 +1,11 @@
+class IdsToBigints20 < ActiveRecord::Migration[5.1]
```

Comment:
The question is, will the stream_entries table survive until the int limit is exhausted?